app/boards/arm/preonic/preonic_rev3.dts

```diff
@@ -131,3 +132,18 @@ zephyr_udc0: &usb {
         };
     };
 };
+
+&layout_ortho_5x12_2x1u {
```

Comment:
Not sure I love this naming, maybe:

```suggestion
&layout_ortho_5x12_all1u {
```

or:

```suggestion
&layout_ortho_5x12_grid {
```

`2x1u` doesn't fully cover the area a 2x2u changes, and is almost defined "in relation to just one of the other layouts" which seems funky. Thoughts? THe `all1u` matches the naming for the 60percent, which makes me favor that slightly.